<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TankerWindow</class>
 <widget class="QDialog" name="TankerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TankerWindow</string>
  </property>
  <widget class="QLabel" name="window_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New/Edit Tanker</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>670</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>670</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fixed</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>On Demand</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="auto_respawn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Respawn</string>
   </property>
  </widget>
  <widget class="QLabel" name="coalition_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Coalition :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="coalition">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neutral</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="pattern_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pattern">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Select Unit</string>
   </property>
  </widget>
  <widget class="QLabel" name="departure_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Departure :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="departure">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Select Departure</string>
   </property>
  </widget>
  <widget class="QLabel" name="parking_size_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parking Size :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="parking_size">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Select Parking</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="airboss_recovery">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Airboss Recovery</string>
   </property>
  </widget>
  <widget class="QComboBox" name="escort">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Select Escort</string>
   </property>
  </widget>
  <widget class="QLabel" name="escort_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Escort Group :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="max_duration">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>120</number>
   </property>
  </widget>
  <widget class="QLabel" name="max_duration_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max mission duration :</string>
   </property>
  </widget>
  <widget class="QLabel" name="altitude_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="altitude">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>500</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="speed_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="speed">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="tacan_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tacan :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tacan">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>126</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="tacan_morse">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="frequency">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="frequency_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency :</string>
   </property>
  </widget>
  <widget class="QLabel" name="fuel_level_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fuel warning level :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fuel_level">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="racetrack_front_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Racetrack front :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="racetrack_front">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="racetrack_back_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>back :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="racetrack_back">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="modex_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modex :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="modex">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="callsign_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Callsign :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="callsign">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Select Callsign</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="callsign_nb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
